sdk/resourcemanager/azure-resourcemanager-storage/src/main/java/com/azure/resourcemanager/storage/models/StorageAccount.java

```diff
@@ -305,13 +317,38 @@
             /**
              * Disables encryption for file service.
              *
+             * @deprecated Azure Storage encryption cannot be disabled.
              * @return he next stage of storage account definition
              */
+            @Deprecated
             WithCreate withoutFileEncryption();
 
+            /**
+             * Specifies that customer-managed key can be enabled for table service.
+             *
+             * Refer to {@link Update#withEncryptionKeyFromKeyVault(String, String, String)} to enable customer-managed
+             * key.
+             *
+             * @return the next stage of storage account definition
+             */
+            WithCreate withTableEncryption();
+
+            /**
+             * Specifies that customer-managed key can be enabled for queue service.
+             *
+             * Refer to {@link Update#withEncryptionKeyFromKeyVault(String, String, String)} to enable customer-managed
+             * key.
+             *
+             * @return the next stage of storage account definition
+             */
+            WithCreate withQueueEncryption();
```

Comment:
This is actually "CAN support customer-managed keys for queues / tables". May need a better name.
https://docs.microsoft.com/en-us/azure/storage/common/customer-managed-keys-overview?toc=/azure/storage/blobs/toc.json#customer-managed-keys-for-queues-and-tables

Reply:
Rename the method to `withTableEncryptionKeyAccountScope` and `withQueueEncryptionKeyAccountScope` to better communicate that it sets the encryption key scope, not whether to encrypt.